### ISSUE TITLE
Vg call error after augment

### ISSUE THREAD

**Issue by Xinyi-Zhang-ZJU** [vgteam/vg#3649]
**1. What were you trying to do?**
mapping short reads to a graph with giraffe and call variants

**2. My command lines were as below, no other error happened before vg call**
```
vg construct -r genome.fa -v PAVs.vcf.gz > graph.vg
vg view graph.vg > graph.gfa
vg autoindex --workflow giraffe -g graph.gfa -p $name -R XG
vg giraffe -Z $name.giraffe.gbz -m $name.min -d $name.dist -f $sample_1.fq -f $sample_2.fq > $sample.gam
vg convert $name.xg > $name.pg
vg augment $name.pg $sample.gam -m 4 -q 5 -Q 5 -A $sample_aug.gam > $sample_aug.pg
vg snarls $name_aug.pg > $name_aug.snarls
vg pack -x $sample_aug.pg -g $sample_aug.gam -o $sample_aug.pack
vg call $sample_aug.pg -r $sample_aug.snarls -k $sample_aug.pack -s $sample > $sample_calls.vcf (error happend)
```
**3. I tried to use $name.giraffe.gbz instead of $name.xg while the same error happened. And I validated the graph/gam as well.**
```
vg validate $sample_aug.pg -a $sample_aug.gam
alignment: valid
graph: valid
```
**4. If you got a line like `Stack trace path: /somewhere/on/your/computer/stacktrace.txt`, please copy-paste the contents of that file here:**

```
Crash report for vg v1.39.0 "Runzi"
Stack trace (most recent call last):
#15   Object "/usr/local/miniconda3/bin/vg", at 0x5b858d, in _start
#14   Object "/usr/local/miniconda3/bin/vg", at 0x1ccaeef, in __libc_start_main
#13   Object "/usr/local/miniconda3/bin/vg", at 0x58c32e, in main
#12   Object "/usr/local/miniconda3/bin/vg", at 0xc7e1db, in vg::subcommand::Subcommand::operator()(int, char**) const
#11   Object "/usr/local/miniconda3/bin/vg", at 0xb6dec8, in main_call(int, char**)
#10   Object "/usr/local/miniconda3/bin/vg", at 0x15bc62e, in virtual thunk to bdsg::PathVectorizableOverlay::for_each_path_handle_impl(std::function<bool (handlegraph::path_handle_t const&)> const&) const
#9    Object "/usr/local/miniconda3/bin/vg", at 0x15ae6f8, in bdsg::PackedPositionOverlay::for_each_path_handle_impl(std::function<bool (handlegraph::path_handle_t const&)> const&) const
#8    Object "/usr/local/miniconda3/bin/vg", at 0x6c349c, in bdsg::BasePackedGraph<bdsg::STLBackend>::for_each_path_handle(std::function<bool (handlegraph::path_handle_t const&)> const&) const
#7    Object "/usr/local/miniconda3/bin/vg", at 0xb6a806, in _ZNSt17_Function_handlerIFbRKN11handlegraph13path_handle_tEEZNS0_20BoolReturningWrapperIZ9main_calliPPcEUlS1_E0_Lb0EE4wrapERKS8_EUlDpOT_E_E9_M_invokeERKSt9_Any_dataS3_
#6    Object "/usr/local/miniconda3/bin/vg", at 0xb690c7, in std::_Function_handler<unsigned long (handlegraph::path_handle_t), main_call(int, char**)::{lambda(handlegraph::path_handle_t)#1}>::_M_invoke(std::_Any_data const&, handlegraph::path_handle_t&&)
#5    Object "/usr/local/miniconda3/bin/vg", at 0x15aebbb, in bdsg::PackedPositionOverlay::get_path_length(handlegraph::path_handle_t const&) const
#4    Object "/usr/local/miniconda3/bin/vg", at 0x15b3854, in bdsg::PagedVector<64ul, bdsg::STLBackend>::get(unsigned long const&) const
#3    Object "/usr/local/miniconda3/bin/vg", at 0x1cdb295, in __assert_fail
#2    Object "/usr/local/miniconda3/bin/vg", at 0x58b727, in __assert_fail_base.cold
#1    Object "/usr/local/miniconda3/bin/vg", at 0x58b857, in abort
#0    Object "/usr/local/miniconda3/bin/vg", at 0x138404b, in raise
```

**5. What does running `vg version` say?**

```
vg version v1.39.0 "Runzi"
Compiled with g++ (Ubuntu 9.4.0-1ubuntu1~20.04) 9.4.0 on Linux
Linked against libstd++ 20210601
Built by stephen@lubuntu
```

**Comment by glennhickey** [comment]
This looks similar to #3629 which was fixed in #3633.  So I think there's a pretty good chance it'll work with the [latest release (1.40)](https://github.com/vgteam/vg/releases/tag/v1.40.0)

**Comment by Xinyi-Zhang-ZJU** [comment]
> 

I appreciate the reply. And I will try the new version as soon.
